.moban.dt/gh-board-gitignore.jj2

```diff
@@ -8,4 +8,5 @@
 *.orig
 *.diff
 
+
```

Comment:
The previously abandoned PR https://github.com/coala/gh-board/pull/70 and my review there https://github.com/coala/gh-board/pull/70#pullrequestreview-136470759 would help you I guess.

Basically, `moban` is a tool which renders your template (`.jj2` files) and generates target files (in this case, `.gitignore`). It then compares the auto-generated file to the real file in the repo. It would complain about any difference (see Travis failure).

It's totally fine if you don't understand `moban`, you can still solve it if you read all the discussions happened in the previous PR carefully.